src/CocktailDetails/CocktailDetails.tsx

```diff
@@ -32,22 +47,15 @@ const CocktailDetails: React.FC<CocktailDetailsProps> = (props) => {
 				<h4>Glass:<div>{cocktailInfo.strGlass}</div></h4>
 				<h4>Instructions:</h4>
 					<p>{cocktailInfo.strInstructions}</p>
-				<h4>Ingredients:</h4>
-					<p>{cocktailInfo.strIngredient1} {cocktailInfo.strMeasure1}</p>
-					<p>{cocktailInfo.strIngredient2} {cocktailInfo.strMeasure2}</p>
-					<p>{cocktailInfo.strIngredient3} {cocktailInfo.strMeasure3}</p>
-					<p>{cocktailInfo.strIngredient4} {cocktailInfo.strMeasure4}</p>
-					<p>{cocktailInfo.strIngredient5} {cocktailInfo.strMeasure5}</p>
-					<p>{cocktailInfo.strIngredient6} {cocktailInfo.strMeasure6}</p>
-					<p>{cocktailInfo.strIngredient7} {cocktailInfo.strMeasure7}</p>
-					<p>{cocktailInfo.strIngredient8} {cocktailInfo.strMeasure8}</p>
-					<p>{cocktailInfo.strIngredient9} {cocktailInfo.strMeasure9}</p>
-					<p>{cocktailInfo.strIngredient10} {cocktailInfo.strMeasure10}</p>
-					<p>{cocktailInfo.strIngredient11} {cocktailInfo.strMeasure11}</p>
-					<p>{cocktailInfo.strIngredient12} {cocktailInfo.strMeasure12}</p>
-					<p>{cocktailInfo.strIngredient13} {cocktailInfo.strMeasure13}</p>
-					<p>{cocktailInfo.strIngredient14} {cocktailInfo.strMeasure14}</p>
-					<p>{cocktailInfo.strIngredient15} {cocktailInfo.strMeasure15}</p>
+				Ingredients:
+				<section className='ingredient-container'>
+					<ul>
+						{displayIngredients(13, 'strIngredient').map(item => <li>{item}</li>)}
+					</ul>
+					<ul className='ingr-measure-list'>
+						{displayIngredients(10, 'strMeasure').map(item => <li>{item}</li>)}
+					</ul>
+				</section>
```

Comment:
This block of code DRY's up the more hardcoded previous block. Makes use of displayIngredients to find all ingredient keys & measure keys to display & map over.

Reply:
Beautiful Becca.